Add note about Docker enhancement (#6341)
DataDog/documentation: content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

### Docker Daemon started after the Host Agent

> If the Docker Daemon starts while the host Agent is already running, restart the Agent to retrigger container collection. This troubleshooting tip only applies to Agent versions prior to 7.17.

Reviewer: ```suggestion
For Agent version prior to 7.17, if the Docker Daemon starts while the host Agent is already running, restart the Agent to retrigger container collection. 
```
